docs/README.md

```diff
@@ -30,23 +30,39 @@ The content in this directory follows the organization of documentation at https
 
 This directory contains the following folders:
 
-\docs <br>
-|--\Enterprise <br>
-|--\actions <br>
-|--\contributing-guide <br>
-|--\data-sources <br>
-|--\how-to <br>
-|--\password-login <br>
-|--\setup <br>
-|--\sso <br>
-|--\tutorial <br>
-|--\widgets <br>
-
-Within these folders, you'll find the Markdown files used for the content. 
+```
+/tooljet/docs
+├── sidebars.json        # sidebar for the next(upcoming) docs version
+├── docs                 # docs directory for the next(upcoming) docs version
+│   ├── Enterprise
+│   │   └── multi-environment.md
+│   └── tooljet-database.md
+├── versions.json        # file to indicate what versions are available
+├── versioned_docs
+│   ├── version-x.x.x    # Current/latest version (set it on docusauras.config.js)
+│   │   ├── Enterprise
+│   │   │   └── multi-environment.md   # https://docs.tooljet.com/docs/Enterprise/multi-environment
+│   │   └── tooljet-database.md.       # https://docs.tooljet.com/docs/tooljet-database
+│   └── version-2.0.0
+│   │   ├── Enterprise
+│   │   │   └── multi-environment.md   # https://docs.tooljet.com/docs/2.0.0/Enterprise/multi-environment
+│   │   └── tooljet-database.md
+│   └── version-1.0.0
+│       ├── Enterprise
+│       │   └── multi-environment.md   # https://docs.tooljet.com/docs/1.0.0/Enterprise/multi-environment
+│       └── tooljet-database.md
+├── versioned_sidebars                 # includes sidebar for the specific versions
+│   ├── version-x.x.x-sidebars.json    
+│   └── version-1.0.0-sidebars.json
+└── src
+│   └── img                           # contains folders that references the images (such as screenshots) used in the \docs\topic
+├── docusaurus.config.js
+└── package.json
+```
 
-\src - contains the ToolJet documentation website code. The documentation website is built using [Docusaurus 2](https://docusaurus.io/).
+`\src` - contains the ToolJet documentation website code. The documentation website is built using [Docusaurus 2](https://docusaurus.io/).
 
-\static\img - contains folders that references the images (such as screenshots) used in the \docs\topic.
+`tooljet\docs\docs\` - This is the base directory and any changes in this directory will reflect on the `next` version the docs. Each of the folders inside it contains markdown files for the corresponding content. 
```

Comment:
This directory serves as the base directory for the documentation. Any modifications made within this directory will be reflected in the next version of the documentation. Each folder inside this directory corresponds to specific content and contains markdown files related to that content.